.github/workflows/torch-inductor/scripts/run_torchinductor_perf.sh

```diff
@@ -14,19 +15,46 @@ sudo nvidia-smi -i 0 -pm 1
 sudo nvidia-smi -i 0 --lock-gpu-clocks=1350,1350
 
 cd "$PYTORCH_DIR" || exit
-TEST_REPORTS_DIR=$TEST_REPORTS_DIR/perf
-mkdir -p "$TEST_REPORTS_DIR"
+TRITON_TEST_REPORTS_DIR=$TEST_REPORTS_DIR/perf
+BASE_TEST_REPORTS_DIR=$TEST_REPORTS_DIR/acc
+mkdir -p "$TRITON_TEST_REPORTS_DIR"
+mkdir -p "$BASE_TEST_REPORTS_DIR"
 
+
+echo "Running with Triton Nightly"
+for model in "${MODELS[@]}"; do
+  if [ "$model" != "$MODEL_SPEC" ] && [ "$MODEL_SPEC" != "all" ]; then
+    continue
+  fi
+  echo "Running performance test for $model"
+  python3 benchmarks/dynamo/"$model".py --float32 -dcuda --training --inductor --performance \
```

Comment:
Do we want to manually disable CUDA Graphs here?

Reply:
I think torchinductor tests both cuda graph and non-cuda graph. The official docs leaves non cuda-graph by default.

Reply:
We do enable more tests without cuda-graph. Timing might be a bit flaky but based on previous runs seems fine.

Reply:
I'm assuming CUDA Graphs are on because there are messages in the log like `skipping cudagraphs due to ['incompatible ops']`.

Reply:
Yeah, that might true. Thanks for reminding me. 
I think the default mode has cudagraph on and is only turned off with `--disable-cudagraphs`.

Reply:
I'll just use whatever documented in README by default